ruby/README.md

```diff
@@ -68,13 +68,23 @@ To Build the JRuby extension, you will need:
 
 First switch to the desired platform with rbenv or RVM.
 
-Then install the required Ruby gems:
+Then install the required Ruby gems (MRI):
 
     $ gem install bundler
     $ bundle
 
+Then install the required Ruby gems (JRuby):
+
+    $ jgem install bundler
```

Comment:
you're right; `jgem` is no longer needed.  I'll remove (running `gem` was hanging, but there was a stuck thread; I reran and it worked fine)